examples/pytorch/language-modeling/train_clm.sh

```diff
@@ -1,19 +1,20 @@
-#!/bin/bash
-LOG_DIR="./logs"
-OUTPUT_DIR="./outputs"
-mkdir -p $LOG_DIR
-mkdir -p $OUTPUT_DIR
-##    SETTINGS     ## 
-MODEL=$1
-BATCH_SIZE=$2
-device_id=$3
-log_file="${LOG_DIR}/${model_name}.log"
-output_dir="$OUTPUT_DIR/$model_name"
+while getopts m:b:g: flag
+do
+    case "${flag}" in
+        m) model=${OPTARG};;
+        b) batch_size=${OPTARG};;
+        g) device_id=${OPTARG};;
+    esac
+done
```

Comment:
While this pattern used for parsing arguments looks nice, I suggest adding default values for these args. 

It should be trivial. Just assign default values before this `while` block.